### PR TITLE
point default local remotesettings file at ion basic add-on on AMO

### PR DESCRIPTION
**Pull request by rhelmer**
@hamilton note that this also changes the XPI URL to point to AMO, if you want to keep it inside the `ion-core-addon` repo that's OK too.

### CODE REVIEW

File: public/locally-available-studies.json
Comment (rhelmer) [inline public/locally-available-studies.json:6]: We are currently using the default icon, this will need to change once we update that.

File: public/locally-available-studies.json
Comment (rhelmer) [inline public/locally-available-studies.json:22]: This and the version number need to change whenever we update the study add-on on AMO. One benefit here though is that we can point it at pre-release versions and test that way before making add-on updates live.

**Comment by rhelmer** [comment]
Hm also before we merge this (assuming you are OK point to AMO) I'd like to push an update to it, since we'll need to update the XPI URL.

**Review by hamilton — APPROVED** [review]

**Pull request merged**
